tests/java-db-test/src/main/java/testing/servlets/TestServlet.java

```diff
@@ -0,0 +1,66 @@
+package testing.servlets;
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.sql.PreparedStatement;
```

Comment:
Order imports alphabetically (`java.sql.PreparedStatement` should be moved down two lines).

Reply:
done